packages/truffle-debugger/lib/controller/sagas/index.js

```diff
@@ -190,12 +190,18 @@ function* stepOver() {
 /**
  * continueUntilBreakpoint - step through execution until a breakpoint
  */
-function* continueUntilBreakpoint() {
+function* continueUntilBreakpoint(action = actions.continueUntilBreakpoint()) {
```

Comment:
This gets run as a listener, right? Seems weird to construct the action here as a default value, when we can expect the type of argument to contain a `breakpoints` key.

```suggestion
function* continueUntilBreakpoint({ breakpoints }) {
```

Reply:
See below.

Reply:
Oh wait nevermind what I was going to write was wrong.  Yeah this would work.  I was just doing redundant defaults like I said.

Reply:
Oh wait I think I know what I was thinking.  It does run as a listener, but I was considering also the possibility that it might be called directly, since some of the others are.

Reply:
My thinking on this is the following:

1. The only interface requirement here is an object with `breakpoints` property that may be undefined.
2. It's extra work to create the action via that function call, and potentially extra cognitive overhead because usually we create actions when putting, not taking.

Reply:
OK. Just, I would also like it to be possible to call it internally with no arguments.  But I can do that without creating an action.

Reply:
Ah right, without arguments at all. That makes the code less pretty. Well still, probably not worth making the action.